src/components/NewMessageForm/SimplePollsEditor/SimplePollsEditor.vue

```diff
@@ -0,0 +1,177 @@
+<!--
+  - @copyright Copyright (c) 2022, Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Modal size="small" v-on="$listeners">
+		<div class="simple-polls-editor">
+			<h2>Create new poll</h2>
+
+			<!-- Poll Question -->
+			<p class="simple-polls-editor__caption">
+				{{ t('spreed', 'Question') }}
+			</p>
+			<InputVue :value.sync="pollQuestion" :placeholder="t('spreed', 'Ask a question ')" v-on="$listeners" />
+
+			<!-- Poll options -->
+			<p class="simple-polls-editor__caption">
+				{{ t('spreed', 'Answers') }}
+			</p>
+			<PollOption v-for="option, index in pollOptions"
+				:key="index"
+				class="poll-option"
+				:value.sync="pollOptions[index]"
+				:placeholder="t('spreed', 'Answer') + ' ' + (index + 1)"
```

Comment:
```suggestion
				:placeholder="t('spreed', 'Answer {option}', {option: index + 1})"
```
is needed for translation compatibility